packages/elements/src/components/ino-snackbar/ino-snackbar.tsx

```diff
@@ -54,11 +55,25 @@ export class Snackbar implements ComponentInterface {
    */
   @Prop() timeout?: number = 5000;
 
+  /**
+   * The role of the dialog. Can be either 'dialog' or 'alertdialog'.
+   * The 'alertdialog' role should be used for important messages that requires user interaction.
+   */
+  @Prop() snackbarRole?: 'alert' | 'alertdialog' = 'alert';
```

Comment:
As a default, can't we derive this state based on the existence of `actionText`?

e.g. Pseudocode
```
snackbarRole = this.snackbarRole ?? this.actionText ? 'alertdialog' : 'alert';
```

Reply:
I am not sure if this is suitable. In general I think we should rethink the default role too, becaue we use the snackbar for notifications that not needs an immediate attention from the user.

From [MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alertdialog_role);

> Warning: Because of its intrusive nature, the alert role must be used sparingly and only in situations where the user's immediate attention is required.

Reply:
Lets implement it like this then:
```
if variant = 'error' || variant = 'warning' && hasAction -> 'alertDialog'
if variant = 'error' || variant = 'warning' -> 'alert'
else 'status'
```